testsuite/forge-test-runner-template.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   serviceAccountName: forge
   containers:
   - name: main
-    image: {FORGE_IMAGE_REPO}:{FORGE_IMAGE_TAG}
+    image: {IMAGE_REPO}/forge:{FORGE_IMAGE_TAG}
```

Comment:
why are we doing this concat here?

Is possible I'd like to move as much logic from templates out

So in that case it would just be a single {IMAGE_LOCATION} and you can do the concat in rust

Reply:
The problem here is that this makes our helm templates contain more logic and it makes it more ingrained

Ultimately we wanna move away from relying on the template implementation and therefore i think it makes more sense to move that kind of logic higher level

so the semantic variables passed in stand on their own, which makes it easier to swap out template implementation, because the template is stupid (think of what variables we might wanna pass to pangu)

Reply:
tru ok ya done